lib/ClangImporter/ImportType.cpp

```diff
@@ -421,6 +421,22 @@ namespace {
       return Type();
     }
 
+    ImportResult VisitDynamicRangePointerType(
+        const clang::DynamicRangePointerType *type) {
+      // DynamicRangePointerType is a clang type representing a pointer with
+      // an "ended_by" type attribute for -fbounds-safety. For now, we don't
+      // import these into Swift.
+      return Type();
+    }
+
+    ImportResult VisitValueTerminatedType(
+        const clang::ValueTerminatedType *type) {
+      // ValueTerminatedType is a clang type representing a pointer with
+      // a "terminated_by" type attribute for -fbounds-safety. For now, we don't
+      // import these into Swift.
+      return Type();
```

Comment:
Same question as above: shouldn't we recurse on the wrapped type here, since it's okay to ignore the `terminated_by` attribute? We'll be gathering this information separately.